proto/spire/server/upstreamca/upstreamca.proto

```diff
@@ -31,11 +31,6 @@ message SubmitCSRRequest {
 }
 
 message SubmitCSRResponse {
-    // Signed certificate (DEPRECATED)
-    bytes DEPRECATED_cert = 1;
-    // Upstream trust bundle (DEPRECATED)
-    bytes DEPRECATED_upstreamTrustBundle = 2;
```

Comment:
Would be good to reserve these indexes so we don't accidentally reuse them in the future